src/tools/tslsplit/Main.hs

```diff
@@ -19,36 +19,73 @@ module Main
 
 import EncodingUtils (initEncoding)
 
-import ArgParseUtils (parseMaybeFilePath)
+import Data.Semigroup ((<>))
+import Options.Applicative
 
 import FileUtils (loadTSL)
 
-import TSL (split, toTSL)
+import TSL (split, splitAssumptions, toTSL)
 
 import System.Directory (getCurrentDirectory)
 
 import System.FilePath (takeBaseName, (<.>), (</>))
 
 -----------------------------------------------------------------------------
 
+-- | The data type contains all flags and settings
+-- that can be adjusted to influence the behavior of the library:
+data Configuration =
+  Configuration
+  { -- | The input file containing the synthesized control flow
+    -- model.
+    inputFile :: FilePath
```

Comment:
In the other tsltools, the input is an optional argument. See e.g. tslsym sources.

Reply:
done in 642955b